scripts/lib/CIME/case.py

```diff
@@ -1395,9 +1395,11 @@ def create(self, casename, srcroot, compset_name, grid_name,
                     logger.warn("Leaving broken case dir {}".format(self._caseroot))
 
             raise
-    def create_clone(self, newcase, keepexe=False, mach_dir=None, project=None, cime_output_root=None,
-                 user_mods_dir=None):
+    def create_clone(self, newcase, keepexe=False, mach_dir=None, project=None,
```

Comment:
It's not clear to me why you added these options to create_clone instead of just setting
EXEROOT and BLDDIR in the _setup section of system_tests_compare_two.py?

Reply:
See the comments I made in the PR:

> I was hoping to do this just via setting EXEROOT and RUNDIR in
system_tests_compare_two, to avoid cluttering up create_clone with
arguments that are just needed for tests. But it turns out that we need
to do this in create_clone, because create_clone calls case_setup, which
creates the EXEROOT and RUNDIR directories.